src/sentry/static/sentry/app/components/organizations/timeRangeSelector.jsx

```diff
@@ -22,11 +22,11 @@ export default class TimeRange extends React.Component {
   }
 
   render() {
-    const {start, end, updateField, runQuery} = this.props;
+    const {className, start, end, updateField, runQuery} = this.props;
```

Comment:
I think renaming: `updateField` --> `onChange` and `runQuery` --> `onUpdate` would make these prop names more general.

Reply:
👍